distributed/stealing.py

```diff
@@ -354,7 +358,7 @@ async def move_task_confirm(self, *, key, state, stimulus_id, worker=None):
                 self.scheduler.total_occupancy -= duration
                 if not victim.processing:
                     self.scheduler.total_occupancy -= victim.occupancy
-                    victim.occupancy = 0
+                    victim.occupancy.clear()
```

Comment:
Coverage complains but on main it is covered. This is very likely an untested path and we're just hitting it implicitly.